nixos/modules/services/matrix/synapse.nix

```
@@ -219,6 +250,34 @@ in {
           '';
         };
 
+        path = mkOption {
+          type = types.nullOr types.path;
+          default = null;
+          description = ''
+            Unix domain socket path to bind this listener to.
+
+            ::: {.note}
+              This option is incompatible with {option}`bind_addresses`, {option}`port`, {option}`tls`
+              and also does not support the `metrics` and `manhole` listener {option}`type`.
```

```suggestion
              and also does not support the `metrics` and `manhole` listener {option}`type`.
            :::
```
Those shouldn't be optional